articles/quickstart/native/ios-swift/04-calling-apis.md

```diff
@@ -17,34 +19,39 @@ budicon: 546
 
 Auth0 provides a set of tools for protecting your resources with end-to-end authentication in your application. 
 
-This tutorial shows you how to get an access token, attach it to a request with an authorization header and call an API. We recommend you use this method for the best security and compliance with RFC standards. 
+In this tutorial, you'll learn how to get a token, attach it to a request (using the authorization header), and call any API you need to authenticate with. 
 
-::: note
-Read more about authentication API on the server-side in [the API documentation](/api/authentication).
-:::
+Before you continue with this tutorial, make sure that you have completed the previous tutorials. This tutorial assumes that:
+* You have completed the [Session Handling](/quickstart/native/ios-swift/03-user-sessions) tutorial and you know how to handle the `Credentials` object.
+* You have set up a backend application as API. To learn how to do it, follow one of the [backend tutorials](/quickstart/backend).
 
-## Get the User's Credentials
+<%= include('../_includes/_calling_api_create_api') %>
 
-You need an access token for your API to check if the request is authenticated. 
+<%= include('../_includes/_calling_api_create_scope') %>
 
-You can retrieve the token from an [Credentials](https://github.com/auth0/Auth0.swift/blob/master/Auth0/Credentials.swift) instance. Read the [Login](/quickstart/native/ios-swift/00-login) article for instructions on how to get credentials.
+## Get the User's Access Token
 
-## Attach the Token
+To retrieve an access token that is authorized to access your API, you need to specify the **API Identifier** value you created in the [Auth0 APIs Dashboard](https://manage.auth0.com/#/apis).
 
-This example shows how to use the `accessToken` value. 
+Present the Hosted Login Page:
 
 ::: note
 Depending on the standards in your API, you configure the authorization header differently. The code below is just an example.
 :::
+let APIIdentifier = API_IDENTIFIER // Replace with the API Identifier value you created
+
+    .audience(APIIdentifier)
```

Comment:
is this `dot` intended?

Reply:
Think this got wrecked during conflict resolution, have restored the snippet.